pytasks.py

```diff
@@ -0,0 +1,50 @@
+"""
+Task2.
+
+pytasks.
+"""
+
+
+def generate_numbers(n=20):
+    """Return a dictionary with square if keys"""
+    import math
+    dic = {}
+    for x in range(n):
+        dic[x + 1] = int(math.pow(x + 1, 2))
+    return dic
+
+
+def count_characters(cnt_str='aaabndlsbfjks'):
+    """Return a dictionary with symbols count in a given line"""
+    dic = {}
+    for x in range(len(cnt_str)):
+        cnt = 0
+        for y in range(len(cnt_str)):
+            if cnt_str[x] == cnt_str[y]:
+                cnt += 1
+        dic[cnt_str[x]] = cnt
+    return dic
+
```

Comment:
tomorrow we will discuss this task and try to find another solution for it. Please, keep in touch with your group and talk about it a bit. 
This solution is good but not "pythonic"